Formula/jenv.rb

```diff
@@ -15,12 +15,23 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+          echo 'status --is-interactive; and source (jenv init -|psub)' >> #{shell_profile}
+          set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME $HOME
```

Comment:
I'm not sure this means the same thing as what I suggested. I'm also unsure about these changes in general since I don't use `fish`, so I'm hoping someone who does can weigh in here 🙂

Reply:
Is there someone else we can ask for a review?

Reply:
Why do we need to check for that variable only in fish?

Reply:
Because the other shells don't seem to require doing anything with `~/.config`